lib/tools/adb-emu-commands.js

```diff
@@ -31,6 +38,26 @@ emuMethods.rotate = async function () {
   await this.adbExecEmu(['rotate']);
 };
 
+emuMethods.powerAC = async function (state = 'on') {
+  if (_.values(emuMethods.POWER_AC_STATES).indexOf(state) === -1) {
+    log.errorAndThrow(`Wrong power AC state sent '${state}', possible values: [${emuMethods.POWER_AC_ON}, ${emuMethods.POWER_AC_OFF}]`);
+  }
+  await this.adbExecEmu(['power', 'ac', state]);
+};
+
+emuMethods.powerCapacity = async function (percent = 100) {
+  percent = parseInt(percent, 10);
+  if (_.isNaN(percent) || percent < 0 || percent > 100) {
+    log.errorAndThrow(`The percentage value should be valid integer between 0 and 100`);
+  }
+  await this.adbExecEmu(['power', 'capacity', percent]);
```

Comment:
I don't think that is necessary. A number should eventually get coerced into a string.